package.json

```diff
@@ -0,0 +1,113 @@
+{
+  "name": "rea-web",
+  "version": "1.0.0",
+  "description": "Web front-end for Registered Email Address Service",
+  "main": "dist/server.js",
+  "scripts": {
+    "build:server": "tsc && cp -r src/views dist/ && cp -r src/config dist/",
+    "build:assets": "gulp build",
+    "build": "npm run build:assets && npm run build:server",
+    "lint": "eslint 'src/*.ts' 'src/**/*.ts' 'test/src/**/*.ts'",
+    "lint:fix": "eslint 'src/*.ts' 'src/**/*.ts' 'test/src/**/*.ts' --fix",
+    "coverage": "nyc --check-coverage --reporter=text-summary npm run test",
+    "coverage:report": "nyc --reporter=lcov --reporter=text npm run test",
+    "test:coverage": "nyc npm test",
+    "test": "mocha --recursive test/src/**/*.ts",
+    "start": "npm run build && source ./dist/config/.env && node dist/server.js",
+    "start:watch": "nodemon"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run lint && npm test"
+    }
+  },
+  "author": "Dale Bradley <dbradley@companieshouse.gov.uk>",
+  "license": "MIT",
+  "dependencies": {
+    "@companieshouse/api-sdk-node": "^2.0.68",
+    "@companieshouse/node-session-handler": "^4.1.9",
+    "@companieshouse/structured-logging-node": "^1.0.8",
```

Comment:
Should this be 'ch-structured-logging-node' and not 'structured-logging-node'? Again, one for @mwejuli-ch probably.

Reply:
to the best of my knowledge, `structured-logging-node` has been deprecated (and archived) in favour of `ch-structured-logging-node` although i can still see a few projects still using the former. i think we should try and use the latter where we can

Reply:
i'll try and add that to the starter actually..

Reply:
@dalebradley - yeah, i couldn't  agree more. the naming structure could have been more consistent.

also, to confirm your point:

`@companiesouse/structured-logging-node` points to `https://github.com/companieshouse/ch-structured-logging-node`.

`npm` confirms it [here](https://www.npmjs.com/package/@companieshouse/structured-logging-node)

Reply:
Good to know. Thanks @mwejuli-ch and @dalebradley .